front/app/components/SignUpIn/SignIn/PasswordSignin.tsx

```diff
@@ -219,7 +233,7 @@ class PasswordSignin extends PureComponent<
       ) {
         try {
           this.setState({ processing: true });
-          const user = await signIn(email, password);
+          const user = await signIn(email, password, rememberMe, expires);
```

Comment:
Why do we need to pass both `remeberMe` and `expires` parameters to the `signIn` method. If feels a little redundant unless I am missing something here?

Reply:
`expires` is the number of days before a cookie is expired. I borrowed this name from `js-cookie` library https://github.com/CitizenLabDotCo/citizenlab/pull/3269/files#diff-d7c40b5b855a6aed7e710e6dad1d4396885615eb05e7436b951134e01d40389cR24
But it seems to be confusing, so I'll rename it.